index.html

```diff
@@ -15,6 +15,23 @@
   </head>
   <body>
     <header>
+      <div class="topBar">
+        <ul class="pages">
+          <li>
+            <a href="#">
+              <i class="fab fa-facebook"></i>
+            </a>
+          </li>
+          <li>
+            <a href="#">
+              <i class="fab fa-twitter"></i>
+            </a>
+          </li>
+          <li><a href="#">English</a></li>
+          <li><a href="#">My Page</a></li>
+          <li><a href="#">Logout</a></li>
+        </ul>
+      </div>
```

Comment:
- [ ] Great job adding the `top dark bar`, But while doing so I notice your `top dark bar` is visible on mobile design. This is not the case according to the provided design. Please make sure the top navbar is hidden on the mobile version. 
- [ ] besides that, I see the previous reviewer asks For the Mobile size (for both the Home And About Page) as stated in the project requirement we need to align the hamburger menu on the left side.
- [ ] Also As stated in the project requirement and as mentioned by the previous reviewer, The mobile menu should have a close (X) button that closes the menu. so Kindly implemented it.
### Give design
![image](https://user-images.githubusercontent.com/69077061/219746059-46022944-93c8-47b9-8d61-efcf620d1995.png)
### Your design 
![Screenshot 2023-02-17 211542](https://user-images.githubusercontent.com/69077061/219746790-5056b5b6-f626-4543-b8f9-69bd263319ea.png)

- [ ] for the menu item, As requested previously. Your design needs to have a list of menus aligned to the right Just like the design with the last item having a `red border`.
- [ ] Also, try to minimize the height of the `top dark bar`, this is to make your design look like the provided design. 
### Give design
![Screenshot 2023-02-17 211230](https://user-images.githubusercontent.com/69077061/219748749-c8ab5754-28a1-47ce-a1a0-5c2687e4202e.png)
### Your design 
![Screenshot 2023-02-17 211335](https://user-images.githubusercontent.com/69077061/219748764-d542a8e8-0a51-4a0b-8034-dd3fd794bf04.png)

Reply:
@bekjon-me , good job removing the `top-bar` from the mobile view but I think you should implement what @TadesseDev suggested about adding the other 3 - 4 navigation links.